yet_another_imod_wrapper/utils.py

```diff
@@ -4,17 +4,25 @@
 import shutil
 from pathlib import Path
 from typing import List, Dict
+from packaging import version
 
 import numpy as np
 
 from yet_another_imod_wrapper.batchruntomo_config.io import write_adoc
 
 
 def imod_is_installed() -> bool:
-    """Check if batchruntomo is on the PATH."""
+    """Check if batchruntomo is on the PATH"""
     return shutil.which('batchruntomo') is not None
 
-
+def imod_version() -> bool:
+    """Check IMOD version is correct."""
+    imod_help_output = os.popen('imod -h').read().split()
+    find_version_idx = imod_help_output.index('Version') + 1
+    imod_version = version.parse(imod_help_output[find_version_idx])
+    correct_imod_version = imod_version >= version.parse('4.11.0')
+    return correct_imod_version, imod_version
```

Comment:
some API feedback
- nice use of `packaging.version`, hadn't seen this before!
- it's not immediately obvious from the function name what it's doing
- `'0.4.11'` is a ['magic string'](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings)
- return type doesn't match returned value

I would instead have this function return the `imod_version` variable and perform the comparison against a constant defined in a `_constants.py` module

Does this make sense?

Reply:
Packaging.version was a useful find - before was using a '.' delimiter on the string then thought there's probs a package for this! Helps me be more lazy. 

Changed the name of the function to extract_imod_version

Have specified required version number in constants to avoid magic! Never trusted magicians anyway